.eslintrc

```diff
@@ -58,6 +58,10 @@ globals:
   # prefs.js
   prefs: false
   setupLivePrefs: false
+  # edit.js
+  exports: false
```

Comment:
No need to expose stuff from edit.js as it's local to editor only. Remove lines 61-64